docs/esm_variables.rst

```diff
@@ -0,0 +1,120 @@
+   setup_name,          Name of the coupled setup.
+   version,             "Version of the model/setup (one of the available options in the ``available_versions`` list)."
+   available_versions,  List of supported versions of the component or coupled setup.
+   git-repository,      Address of the model's git repository.
+   branch,              Branch from where to clone.
+   destination,         "Name of the folder where the model is download and compiled, in a coupled setup."
+   comp_command,        Command used to compile the component.
+   executable,          Name of the component executable file.
+
+Runtime variables
+-----------------
+.. csv-table::
+   :header: Key, Description
+   :widths: 15, 85
+
+   account,             User account to be used to run the experiment.
+   model_dir,           "Absolute path of the model directory (where it was installed by `esm_master`)."
+   setup_dir,           "Absolute path of the setup directory (where it was installed by `esm_master`)."
+   compute_time,        "Estimated computing time for a run, used for submitting a job with `slurm`."
+   time_step,           Time step of the component in seconds.
+   lresume,             Boolean to indicate whether the run is an initial run or a restart.
+   pool_dir,            Path to the pool directory.
+   namelists,           "List of namelist files required for the model."
+   namelist_changes,    "Functionality to handle changes in the namelists from the yaml files (see :ref:`yaml:Changing Namelists`)."
+   nproc,               Number of processors to use for the model.
+   nproca/nprocb,       "Number of processors for different MPI tasks/ranks. Incompatible with ``nproc``."
+   base_dir,            Path to the directory that will contain the experiment folder (where the experiment will be run and data will be stored).
+   post_processing,     Boolean to indicate whether to run postprocessing or not.
+   ":ref:`yaml:File Dictionaries`",     "`YAML` dictionaries used to handle input, output, forcing, logging, binary and restart files (see :ref:`yaml:File Dictionaries`)."
+   expid,               "ID of the experiment, also defined when calling ``esm_runscripts`` with the ``-e`` flag."
+   ini_restart_exp_id,  "ID of the restarted experiment in case the current experiment has a different ``expid``."
+   ini_restart_dir,     "Path of the restarted experiment in case the current experiment runs in a different directory."
+   execution_command,   "Command for executing the component, including ``${executable}`` and the necessary flags."
+
+Calendar variables
+------------------
+.. csv-table::
+   :header: Key, Description
+   :widths: 15, 85
+
+   initial_date,        "Date of the beginning of the **simulation** in the format YYYY-MM-DD. If the simulation is a restart, ``initial_date`` marks the beginning of the restart."
```

Comment:
Is somewhere described what the difference between 'simulation' and 'run' is?

Reply:
It is kind of implicitly defined in the experiment folder structure section, do you think that is enough?

Reply:
Yes, I think we don't need to explain it here again.